kotlin/workflow-runtime/src/main/java/com/squareup/workflow/testing/WorkflowTestParams.kt

```diff
@@ -59,9 +58,30 @@ data class WorkflowTestParams<out StateT>(
     /**
      * Starts the workflow from its initial state (as specified by
      * [initial state][com.squareup.workflow.StatefulWorkflow.initialState]), with a non-null
-     * snapshot.
+     * snapshot.  Only applies to [StatefulWorkflow][com.squareup.workflow.StatelessWorkflow]s.
+     *
+     * This differs from [StartFromCompleteSnapshot] because it represents only the snapshot for
+     * the root workflow, without any of the snapshots of its children or other bookkeeping data
+     * added by the workflow runtime.
+     *
+     * @param snapshot A [Snapshot] that can be directly parsed by a workflow's `initialState`
+     * method. For workflow trees, this is only the snapshot of the _root_ workflow. Testing with
+     * a complete snapshot of a workflow tree is not currently supported.
```

Comment:
Maybe say explicitly that this is the snapshot returned by the root's `StatefulWorkflow.snapshotState`, for easier contrast with the `WorkflowTester.awaitNextSnapshot` bit below.